packages/listview/src/components/AuditLogs/filterDropdown/filters.tsx

```diff
@@ -0,0 +1,130 @@
+import React, { useContext, useRef } from 'react';
+import { Option, ButtonIcon } from 'react-rainbow-components';
+import { TrashFilled } from '@rainbow-modules/icons';
+import { StyledFilterContainer, StyledLabel, StyledPicklist, StyledInput } from './styled';
+import { LabelFilter as LabelFilterType, PicklistValue } from '../types';
+import context from '../context';
+import getLabelFilters from './helpers/getLabelFilters';
+
+const FilterOptions = ({ labels = [] }: { labels: string[] }) => (
+    <>
+        {labels.map((label) => (
+            <Option name={label} label={label} />
+        ))}
+    </>
```

Comment:
here you can return the map directly, then remove the fragment

Reply:
This is no possible to achieve due to current limitations of TypeScript and Rect typings.

See [here](https://github.com/DefinitelyTyped/DefinitelyTyped/issues/20356).